module/configs/pych.json

```diff
@@ -102,7 +102,7 @@
             ],
 
             "commands": [
-                "chpl %(cflags)s %(ctargets)s -o __TMP_PATH__",
+                "chpl %(cflags)s __DEPENDS__ %(ctargets)s -o __TMP_PATH__",
```

Comment:
Its use in the "commands" is replaced during the compiler initialization with the contents of the "cflags" variable, so by the time materialization rolls around, it isn't visible in the command.  Since I didn't want to update the command globally, I did this instead.